samples/gltf_viewer.cpp

```diff
@@ -946,25 +965,27 @@ int main(int argc, char** argv) {
         if (view->getStereoscopicOptions().enabled) {
             Camera& c = view->getCamera();
             auto od = app.viewer->getOcularDistance();
-            // Eye 0 is always rendered to the left side of the screen; Eye 1, the right side.
+            // Eyes are rendered from left-to-right, i.e., eye 0 is rendered to the left side of the
+            // window.
             // For testing, we want to render a side-by-side layout so users can view with
             // "cross-eyed" stereo.
             // For cross-eyed stereo, Eye 0 is really the RIGHT eye, while Eye 1 is the LEFT eye.
             const mat4 rightEye = mat4::translation(double3{ od, 0.0, 0.0});    // right eye
             const mat4 leftEye  = mat4::translation(double3{-od, 0.0, 0.0});    // left eye
-            c.setEyeModelMatrix(0, rightEye);
-            c.setEyeModelMatrix(1, leftEye);
-            mat4 projections[2];
-            // Use an aspect ratio of 1.0. The viewport will be taken into account in
-            // FilamentApp.cpp.
-            projections[0] = mat4::perspective(70.0, 1.0, .1, 10.0);
-            projections[1] = mat4::perspective(70.0, 1.0, .1, 10.0);
-            c.setCustomEyeProjection(projections, 2, projections[0], .1, 10.0);
-            // FIXME: the aspect ratio will be incorrect until configureCamerasForWindow is
-            // triggered, which will happen the next time the window is resized.
+            const mat4 modelMatrices[4] = { rightEye, leftEye, rightEye, leftEye };
+            for (int i = 0; i < std::min(app.config.stereoscopicEyeCount, 4); i++) {
```

Comment:
nit: there's a mismatch here with `CONFIG_MAX_STEREOSCOPIC_EYES` if say `CONFIG_MAX_STEREOSCOPIC_EYES` needs to raised to 6 in the future.  Maybe we should just use `4` in the `std::cerr` above?

Reply:
Yeah, good point. In fact I can simply make "even" eyes be the right eye and "odd" eyes be the left eye. That way, this will always work no matter the eye count.